src/OMSimulatorLib/ComponentFMUCS.cpp

```diff
@@ -667,6 +667,30 @@ oms_status_enu_t oms::ComponentFMUCS::getBoolean(const fmi2_value_reference_t& v
   return oms_status_ok;
 }
 
+/**
+ * @brief Get boolean value(s) of FMU component.
+ *
+ * @param vr                      Array with value references.
+ * @param value                   Array containing values on output.
+ * @param nvr                     Length of arrays vr and value.
+ * @return oms_status_enu_t       Return oms_status_ok on success.
+ */
+oms_status_enu_t oms::ComponentFMUCS::getBoolean(const fmi2_value_reference_t vr[], bool value[], size_t nvr)
+{
+  CallClock callClock(clock);
+
+  int stupidIntBoolArray[nvr];
```

Comment:
MSVC doesn't like this:
```
       "c:\dev\jenkins\ws\OMSimulator_PR-993\build\win\INSTALL.vcxproj" (Build target) (1) ->
       "C:\dev\jenkins\ws\OMSimulator_PR-993\build\win\ALL_BUILD.vcxproj" (default target) (3) ->
       "C:\dev\jenkins\ws\OMSimulator_PR-993\build\win\src\OMSimulatorLib\OMSimulatorLib.vcxproj" (default target) (5) ->
       (ClCompile target) -> 
         C:\dev\jenkins\ws\OMSimulator_PR-993\src\OMSimulatorLib\ComponentFMUCS.cpp(682): error C2131: expression did not evaluate to a constant [C:\dev\jenkins\ws\OMSimulator_PR-993\build\win\src\OMSimulatorLib\OMSimulatorLib.vcxproj]


       "c:\dev\jenkins\ws\OMSimulator_PR-993\build\win\INSTALL.vcxproj" (Build target) (1) ->
       "C:\dev\jenkins\ws\OMSimulator_PR-993\build\win\ALL_BUILD.vcxproj" (default target) (3) ->
       "C:\dev\jenkins\ws\OMSimulator_PR-993\build\win\src\OMSimulatorLib\OMSimulatorLib_static.vcxproj" (default target) (6) ->
         C:\dev\jenkins\ws\OMSimulator_PR-993\src\OMSimulatorLib\ComponentFMUCS.cpp(682): error C2131: expression did not evaluate to a constant [C:\dev\jenkins\ws\OMSimulator_PR-993\build\win\src\OMSimulatorLib\OMSimulatorLib_static.vcxproj]
```
Is this a new C++ feature? Can someone explain this?

Any way I'll change `value[]` to be an int array which fmi is expecting so we don't allocate a new array for each call of getBoolean.